src/mon/MonCommands.h

```diff
@@ -444,6 +444,12 @@ COMMAND("fs rename "
 	"name=new_fs_name,type=CephString,goodchars=" FS_NAME_GOODCHARS
 	" name=yes_i_really_mean_it,type=CephBool,req=false",
 	"rename a ceph file system", "mds", "rw")
+COMMAND("fs swap "
+	"name=fs1_name,type=CephString "
+	"name=fs2_name,type=CephString "
+	"name=swap_fscids,type=CephBool,req=false "
```

Comment:
What's the default - not to swap fscids?

Reply:
Yes. Patrick suggested it - https://github.com/ceph/ceph/pull/50212#discussion_r1116200184

Reply:
Perhaps turn it into a required option with `yes|no` so that the operator must think about it.